src/60_Samples_%26_Templates/Housing.html

```diff
@@ -126,51 +153,55 @@
     <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
     </head>
     <body>
-            <!-- ## Heading -->
-            <!--[amp-fit-text](https://www.ampproject.org/docs/reference/extended/amp-fit-text.html) can be used to fit the text to the available space. It works well for titles that vary not too much in length. -->
-        <h4 class="mx1">
+        <!-- ## Heading -->
+        <!--[amp-fit-text](https://www.ampproject.org/docs/reference/extended/amp-fit-text.html) can be used to fit the text to the available space. It works well for titles that vary not too much in length. -->
+        <div>
+          <h4>
```

Comment:
Can you change h4 to h1 as it's the main heading of the page.

Reply:
done